packages/store/ts/config/parseStoreConfig.ts

```diff
@@ -170,45 +200,58 @@ export const zEnumsConfig = z.object({
  *    FINAL
  *
  ************************************************************************/
+// Manual *UserConfig is used instead of `z.input<typeof zStoreConfig>` because zod can't preserve doc comments
 
-// zod doesn't preserve doc comments
-export type StoreUserConfig<
-  EnumNames extends StringForUnion = StringForUnion,
-  StaticUserTypes extends ExtractUserTypes<EnumNames> = ExtractUserTypes<EnumNames>
-> = EnumsConfig<EnumNames> & {
+/** Options without generics */
+export interface StoreSimpleOptions {
```

Comment:
why the separation into its own type?

Reply:
for `typeExtensions`, but I actually don't need it and can just use `z.input<typeof zStoreConfig>`. Fixed